clientv3/op.go

```diff
@@ -145,10 +157,22 @@ func WithPrefix() OpOption {
 		op.end = []byte{0}
 	}
 }
+
+// WithRange specifies the range of 'Get' or 'Delete' requests.
+// For example, 'Get' requests with 'WithRange(end)' returns
+// the keys in the range [key, end).
 func WithRange(endKey string) OpOption {
 	return func(op *Op) { op.end = []byte(endKey) }
 }
+
+// WithFromKey specifies the range of 'Get' or 'Delete' requests
+// that keys are greater than or equal to key in the argument.
 func WithFromKey() OpOption { return WithRange("\x00") }
+
+// WithSerializable serializes 'Get' requests. Which are
```

Comment:
WithSerializable makes 'Get' request serializable. By default, it's linearizable. Serializable requests are better for lower latency requirement.